### PR TITLE
task 9

### PR DESCRIPTION
**Pull request by Brums16**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -434,3 +434,25 @@ describe("PATCH /api/articles/:articleid/", () => {
...
+describe("DELETE /api/comments/:comment_id", () => {
+  test("responds with status code 204", () => {
+    return request(app).delete("/api/comments/6").expect(204);
+  });
+  test("responds with status 404 and appropriate error message if comment id not found", () => {
+    return request(app)
+      .delete("/api/comments/3000")
+      .expect(404)
+      .then(({ text }) => {
+        expect(text).toBe("No comment found for comment_id 3000");
```
Comment (xViolaine) [inline __tests__/app.test.js:447]: I like the message, I would probably refrain from literally calling it "comment_id", as usually it's best practice to not mention what our data looks like internally/in our tables :D Only a small thing

File: models/topics-model.js [hunk 2]
```diff
@@ -101,3 +101,18 @@ exports.updateArticle = async (inc_votes, id) => {
...
+      DELETE FROM comments WHERE comment_id = $1 RETURNING *
+     `,
+    [id]
+  );
+  if (rows.length === 0) {
+    return Promise.reject({
+      status: 404,
+      msg: `No comment found for comment_id ${id}`,
+    });
+  }
```
Comment (xViolaine) [inline models/topics-model.js:117]: Always a bit exciting to see async await!
Well done :)

**Review by xViolaine — APPROVED** [review]
Looks good, nicely done!

**Comment by Brums16** [comment]
Thanks for the review, will update the comment_id and some other id's I've got in error messages too!

**Pull request merged**